EIPS/eip-2462.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: 2462
+title: Interface standard for Ethereum compatibles
+author: Jet Lim (@daoauth), sooyoung hyun (@hsy822)
+discussions-to: https://github.com/ethereum/EIPs/issues/2463
+status: Draft
+type: Standards Track
+category: Interface
+created: 2020-01-13
+---
+
+## Simple Summary
+Interface standard for transaction transfer between blockchain networks compatible with Ethereum
+
+## Abstract
+This EIP is a document that defines a transaction relay standard that applies meta-tx between Ethereum and compatible chains with the same addressing system.
+
+## Motivation
+Just as having the same private key creates the same address (like Ethereum mainnet and testnet), Ethereum-compatible chains developed by forking Ethereum will generate the same address from the same private key.
+
+The process of verifying the address of a transaction sender by verifying a signed transaction in each wallet of an Ethereum-compatible chain with the same private key is also the same.
+
+Similarly, Ethereum-compatible chains can use the same smart contracts as Ethereum, and operate in the same way but with much faster TPS, with a few gas.
+
+Therefore, the process of executing smart contracts by delegating gas bills to relays using the signed meta-tx in the standard Ethereum wallet is the same on the Ethereum mainnet and the Ethereum-compatible chain.
+
+With this in mind, the Ethereum mainnet, a public chain, is used for popular purposes such as DID registry and verification. And Ethereum-compatible chains, which have the advantages of high TPS (!) And low gas cost (!), Can be used as side chains to quickly execute smart contract functions.
+
+And with Ethereum standard wallets, you can implement high-availability services that operate by delegating gas to service providers with minimal development costs.
+
+
+## Use Case
```

Comment:
Use case isn't a standard section for EIPs.  Instead, there should be a `## Specification` section that defines the requirements of implementations on both sides of the protocol.